### PR TITLE
Develop/arceveti/general 2

### PR DESCRIPTION
**Pull request by KazeEmanuar**
hubgpgbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb yip file

### CODE REVIEW

**Comment by thecozies** [comment]
> hubgpgbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb yip file

while i agree, i think we should reconsider

**Comment by arthurtilly** [comment]
this is a very important matter, thank you kaze emanuar

**Comment by Arceveti** [comment]
merge conflict

File: README.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ This is a fork of the ultrasm64 repo by CrashOveride which includes the followin
 - **aglab2**: Bugfixes (particularly puppycam), refactor stuff
 - **someone2639**: math.s and crash screen disam, stack trace, map packing, shiftable segments 2, S2DEX engine
 - **Arthurtilly**: Platform Displacement 2
-- **Fazana**: PuppyLib, ucode swapping, audio load time optimisations (with Arctic), general hacker qol improvements, visual debug
+- **Fazana**: PuppyLib, ucode swapping, audio load time optimizations (with Arctic), general hacker qol improvements, visual debug
```
Comment (FazanaJ) [inline README.md:21]: Minor spelling mistake

**Comment by axollyon** [comment]
This NEEDS to be on PC Port!

**Comment by 12abz** [comment]
> This NEEDS to be on PC Port!

pp port is trash.

**Comment by 12abz** [comment]
red, you know i'm not wrong.

**Pull request closed without merging**